### PR TITLE
Add Phaser-CE docset

### PR DESCRIPTION
**Pull request by ArtBIT**
Adding docset for Phaser - Community Edition https://github.com/phaserjs/phaser-ce

### CODE REVIEW

File: docsets/Phaser-CE/docset.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Phaser-CE",
+  "version": "2.20.1",
+  "archive": "Phaser-CE.tgz",
```
Comment (Kapeli) [inline docsets/Phaser-CE/docset.json:4]: This file doesn't exist.

Comment (ArtBIT) [reply docsets/Phaser-CE/docset.json:4]: I thought I had to append the .txt to the actual file (Phaser-CE.tgz.txt), but omit the .txt here?

**Comment by Kapeli** [comment]
Please follow the format at https://github.com/Kapeli/Dash-User-Contributions/tree/master/Sample_Docset. The `txt` file is something my processing leaves behind after pushing a new archive to the CDN.

**Comment by Kapeli** [comment]
Failed validation: some entries in your `docSet.dsidx` have an empty name set.

**Comment by Kapeli** [comment]
Thank you!

**Comment by ArtBIT** [comment]
Thank *you* for your patience. All the best.

**Pull request merged**